### ISSUE TITLE
Works perfectly in vanilla Node.js, but returns no results in React Native

### ISSUE THREAD

**Issue by atljoseph** [krisk/Fuse#208]
SAMPLE NODE SCRIPT: (works)

const Fuse = require('fuse.js');
const pathTotags = 'value';
const config = {
	id: pathTotags,
	shouldSort: true,
	threshold: 0.6,
	location: 0,
	distance: 100,
	maxPatternLength: 32,
	minMatchCharLength: 1,
	keys: [pathTotags],
};
const list = [
        { value: 'ajnckajfn' },
		{ value: 'ajnsdfasldkckajfn' },
		{ value: 'o;ip09ksd' },
		{ value: ' xkln' },
		{ value: 'uwerh' },
		{ value: 'sdk;fmsdl;km' },
    ];
	
const fuse = new Fuse(list, config);
const result = fuse.search('a');
console.log(result);

// console.log('help');

I have this set up in React Native in the same way with no results returned, but also no errors, either.

**Comment by atljoseph** [comment]
always returns an empty array

**Comment by atljoseph** [comment]
ah, it is because it was inside of a mobX class. evidently it doesn't work with the observableArray there

**Comment by atljoseph** [comment]
for anyone else with this issue, use this: https://mobx.js.org/refguide/tojson.html

**Comment by Sigmus** [comment]
I believe you may close this issue if everything has been sorted out.

**Comment by ashleshsortee** [comment]
It is not working @atljoseph

**Comment by jonlaing** [comment]
I just ran into this, and it was a result of me screwing up, so I figured I'd put it here for posterity in case anyone else runs into this.

I was using Redux to fetch my items, but was instantiating the Fuse search in `componentDidMount`. That meant when it got instantiated, redux hadn't finished populating all of my items, so Fuse was searching on an empty list. The solution was to make sure that I reinstantiated Fuse when the component got new items from Redux in `componentDidUpdate`.

One of those silly things you're bound to forget when prototyping quickly. Hope this helps someone.
